src/lib/y2network/interface_config_builder.rb

```diff
@@ -475,5 +452,32 @@ def original_hostname
     def driver_auto?
       :auto == driver
     end
+
+    # Converts aliases in hash form to a list of IPConfig objects
+    #
+    # @return [Array<IPConfig>]
+    def aliases_to_ip_configs
+      last_id = 0
+      used_ids = aliases.select { |a| a[:id] && a[:id] =~ /\A\d+\z/ }.map { |a| a[:id].to_i }
+      aliases.each_with_object([]) do |map, result|
+        ipaddr = IPAddress.from_string(map[:ip])
+        ipaddr.prefix = map[:prefixlen].delete("/").to_i if map[:prefixlen]
+        id = map[:id]
+        last_id = id = find_free_alias_id(used_ids, last_id) if id.nil? || id.empty?
+        result << ConnectionConfig::IPConfig.new(ipaddr, label: map[:label], id: id.to_s)
+      end
+    end
+
+    # Returns a free numeric ID for an IP aliases
+    #
+    # @param used_ids   [Array<Integer>] Already used IDs
+    # @param current_id [Integer] Current used ID
+    def find_free_alias_id(used_ids, last_id)
+      loop do
```

Comment:
btw with new rubocop we can use `(last_id..)` but not with current one, as it is supported in new version.